docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -39,7 +39,7 @@ A devfile allows to specify commands to be available for execution in a workspac
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
```

Comment:
@maysunfaisal, thanks. quick questions after looking at your points:
1. When you say, `there are more than two types of commands` do you want me to include other commands? 
5. I do not know much about `command preview URL`. Looking at the schema, I see that everything under `commands` is correct. But I wouldn't know how to expand it since it's not making sense. Do you think we should remove it?